src/containers/Chat/ChatMessages/ChatMessages.tsx

```diff
@@ -287,7 +287,9 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({ contactId }) => {
   }
 
   let messageList: any;
-  if (conversations.messages.length > 0) {
+
+  // Must have a contact + conversations in order for messages to render.
```

Comment:
didn't get this comment completely. When we say "contact", what are we referring to exactly inside the if statement?